services/issue/issue.go

```diff
@@ -130,6 +130,15 @@ func UpdateAssignees(issue *issues_model.Issue, oneAssignee string, multipleAssi
 	return err
 }
 
+func DeleteIssues(doer *user_model.User, gitRepo *git.Repository, issues []*issues_model.Issue) error {
+	for _, issue := range issues {
+		if err := DeleteIssue(doer, gitRepo, issue); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
Hmm, we could think about passing a context in as well and executing this whole block inside a transaction so that it is automatically rolled back if the deletion of any issue fails.